interface/billing/sl_eob_search.php

```diff
@@ -232,8 +232,9 @@ function upload_file_to_client_pdf($file_to_send) {
   
   
     // Print or download statements if requested.
-    //
-  if (($_POST['form_print'] || $_POST['form_download'] || $_POST['form_pdf']) || $_POST['form_portalnotify'] && $_POST['form_cb']) {
+    // This code will only be ran IF form_cb has AT LEAST 1 checkbox selected
+
+    if (($_POST['form_print'] || $_POST['form_download'] || $_POST['form_pdf'] || $_POST['form_portalnotify']) && isset($_POST['form_cb'])) {
```

Comment:
Moving form_portalnotify into the form checking should be the only fix that is required. The isset will take care of a php 7.1 warning.

Reply:
Should I only remove "isset" ?

Reply:
no leave it .It will have to be addressed by who ever does the fixes for 7.1 compatibility.

Reply:
No changes necessary then?